### PR TITLE
Fix/be(31) google auth

### PR DESCRIPTION
**Pull request by Gathu17**
# General Changes

-   Fixed the google auth response to return token

## Developer Notes

---

## Checklist

-   [ ] The base branch is set to `dev`
-   [ ] The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
-   [ ] The Linear issue has been linked to the PR
-   [ ] The General Changes section has been filled out
-   [ ] Developer Notes have been added (optional)
-   [ ] End-to-end tests(if any) are passing without any errors
-   [ ] Code style generally follows existing patterns
-   [ ] New third-party packages, if any, do not introduce potential security threats
-   [ ] There are no CI changes, or they have been OK’d by the devops team
-   [ ] Code changes have been quality checked in the ephemeral URL
-   [ ] Errors are handled using the right error handles
-   [ ] The right thing is returned

### CODE REVIEW

**Comment by Delight-Fela-Steve** [comment]
Please kindly check for leaked seceret keys

**Comment by Gathu17** [comment]
> Please kindly check for leaked secret keys

The secrets leak has been resolved.

**Comment by Delight-Fela-Steve** [comment]
Kindly check the typo error in the url for the passport redirect

**Comment by Gathu17** [comment]
> Kindly check the typo error in the url for the passport redirect

Done

**Review by Delight-Fela-Steve — APPROVED** [review]

File: server/routes/authRoutes.js [hunk 1]
```diff
@@ -26,10 +27,12 @@ router.get('/google',
   passport.authenticate('google', { scope : ['profile', 'email'] }));
  
 router.get('/google/callback', 
-  passport.authenticate('google', { failureRedirect: '/error' }),
+  passport.authenticate('google', { failureRedirect: 'coverly.hng.tech/sigin' }),
```
[the remark this answers is not in the document]
Comment (Gathu17) [reply server/routes/authRoutes.js]: check latest commit

**Pull request merged**